packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts

```diff
@@ -105,9 +109,19 @@ class DataStorageAuthWithTriggerTestProject extends TestProjectBase {
   /**
    * @inheritdoc
    */
-  override async deploy(backendIdentifier: BackendIdentifier) {
+  override async deploy(
+    backendIdentifier: BackendIdentifier,
+    environment: Record<string, string> = {}
```

Comment:
Personally I would rather remove this prop entirely here and just generate the env object internally, but this can be easily refactored later if needed.